src/utils/teamHelper.js

```diff
@@ -1,13 +1,21 @@
-const nconf = require('nconf');
+const getTeams = config => {
+  let teams = {};
+  const teamsJSON = config.get('teams') || {};
 
-const teamsJSON = nconf.get('teams') || {}; // add a default object if none
+  try {
+    debugger;
```

Comment:
Probably remove this `debugger` statement.

Reply:
:yikes: